scrambles/src/main/java/net/gnehzr/tnoodle/scrambles/Main.java

```diff
@@ -1,89 +0,0 @@
-package net.gnehzr.tnoodle.scrambles;
```

Comment:
I'm ok with getting rid of this, but for the record, I think some people do use this to use TNoodle to generate scrambles in a headless way. We'll probably hear from the eventually. It would be nice for us to have a good answer for them when they do contact us =)

Reply:
The best answer will most likely be to use the webserver API in conjunction with any REST client that exists out there.
Possible answers are:
- `Insomnia` on Linux
- `Paw` on macOS
- `Don't use Windows(TM)` on Windows